cli/src/stake.rs

```diff
@@ -2501,6 +2521,19 @@ pub fn process_delegate_stake(
     }
 }
 
+pub fn process_stake_minimum_delegation(
+    rpc_client: &RpcClient,
+    _config: &CliConfig, // bprumo TODO: am I supposed to use the config somewhere?
```

Comment:
nah. this is fine.  `CliConfig` is mostly just a way to squirrel global arg values away.  since you aren't signing, no need to use (or even pass) it

Reply:
> I don't think setting the commitment level to a different value makes sense for the RPC call.

I actually disagree here. The RPC method takes a commitment, because at an epoch boundary where the minimum delegation changes, the processed vs confirmed vs finalized banks will return different values.
It's not really a big deal, since it only affects a handful of slots before everything is consistent again, but I think this should use `CliConfig` in order to not be confusing at such an epoch boundary.

Reply:
> I could also make a CliStakeMinimumDelegation struct in cli_output.rs and implement QuietDisplay/VerboseDisplay/Display on it. That seems like overkill for just a u64 for lamports. Thoughts?

I don't feel super strongly about the opinion to follow, since no one has complained about `solana balance`, but I sort of want all the cli commands to implement json output, which would require some sort of struct. It could be a generic balance display struct. Wdyt? Wouldn't need to be part of this PR.

Reply:
> I sort of want all the cli commands to implement json output

an unstructured number is valid JSON :wink: 
```
$ echo 1000000000 | jq
1000000000
```

Reply:
> Yes, but what do we return if the user passes --lamports? :wink:

```
1000000000 lamports
```

which is _not_ valid JSON :wink: 

```
$ echo "1000000000 lamports" | jq
1000000000
parse error: Invalid numeric literal at line 2, column 0
```

so the lazy solution here is to just use `encoding == {json,json-compact}` to control the units output of `build_balance_message`

Reply:
That's certainly an option. Feel free to comment that on the issue Brooks just created :)

Reply:
ah, missed that there was an issue for this too :man_facepalming: